config/nrfconnect/chip-module/Kconfig

```diff
@@ -76,6 +76,11 @@ config CHIP_BOOTLOADER_NONE
 
 endchoice
 
+choice SUIT_DFU_CANDIDATE_PROCESSING_LEVEL
```

Comment:
This is a minor, but wouldn't make more sens to do:

if CHIP_BOOTLOADER_SUIT
choice SUIT_DFU_CANDIDATE_PROCESSING_LEVEL
default SUIT_DFU_CANDIDATE_PROCESSING_PUSH_TO_CACHE
endchoice
endif

?

Reply:
Hmm, ok, let's do it this way, so it will not be visible for other samples.